messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "Hallo.",
+        "Sicher.",
+        "Nein",
+        "Segmentierungsfehler",
+        "Segmentation fault (core dumped)",
+        "HTTP-Fehler 418: Ich bin eine Teekanne",
+        "🐺",
+        "Ich bin hier!",
+        "Ich weigere mich!",
+        "Was?",
+        "Meinst du nicht {=ping!command:!:bold}?",
+        "{bot_nick}.exe hat aufgehört zu arbeiten. Windows sucht nach einer Lösung für das Problem...",
+        "skynet.exe hat aufgehört zu arbeiten. Windows sucht nach einer Lösung für das Problem...",
+        "1337 Sekunden lang kein Ping empfangen.",
+        "Herzlichen glückwunsch! Sie sind die 1337. Person, die {=ping!command:!} verwendet. Sie gewinnen eine Ziege!\n{nick:@}'s Ziege geht vorbei und macht eine Kopfnuss bei {nick:@}",
+        "PING! {nick}",
+        "PONG! {nick}",
+        "Es tut mir leid Dave, ich fürchte, ich kann das nicht.",
+        "Gib mir einen Ping, Vasily. Nur ein Ping, bitte.",
+        "Würden Sie bitte mich in Ruhe lassen? Ernsthaft."
+    ],
+    "gifs": [
+        "https://i.imgur.com/nO8rZ.gifv",
+        "https://i.imgur.com/uGVfZ.gifv",
+        "https://i.imgur.com/VvsgyXJ.gifv",
+        "https://i.imgur.com/PIIfL15.gifv",
+        "https://i.imgur.com/nly0Cmm.gifv",
+        "https://i.redd.it/dstee514i5zz.jpg",
+        "https://i.imgur.com/jWFy9aV.gifv",
+        "https://i.imgur.com/aK34qpZ.gifv"
+    ],
+    "latency": "{0:.3f} Sekunde(en).",
+    "lynch_reveal": [
+        "Die Dorfbewohner entscheiden nach vielen Debatten endlich über das Lynchen von {0:@} und {0:@} entpuppte sich als {1!role:article} {1!role:bold}.",
+        "Eine Abstimmung ist getroffen wurden, und die Dorfbewohner lynchten {0:@}, {1!role:article} {1!role:bold}.",
+        "Nach langem Ringen wird  {0:@} zum Galgen gebracht und nach dem Tod wurde entdeckt das {0:@} {1!role:article} {1!role:bold} war.",
+        "Die Dorfbewohner entscheiden sich {0:@} zu hängenfffff das Seil dehnt sich und reist, und der folgende Sturz tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner haben es schwer mit dem Schmerz des Todes von {0:@} und lynchten wiederwillig {1!role:article} {1!role:bold}.",
+        "Konform mit dem Willen des Dorfes, erwiesen sich die Galgen als wirksam bei der Tötung von {0:@}, {1!role:article} {1!role:bold}.",
+        "In einem Anfall von Hysterie lynchen die Dorfbewohner {0:@}, und töten {1!role:article} {1!role:bold}.",
+        "Während sie auf ihre Hinrichtung warten, fällt ein Baum auf {0:@}, dies tötet {1!role:article} {1!role:bold}.",
+        "Die Dorfbewohner fesseln {0:@}. Und werfen {0:@} in einen See. Es scheint so das {1!role:article} {1!role:bold} nicht gut schwimmen kann",
+        "Glaubt man ihrem Nachbarn und Freund das {0:f@} gefährlich zu sein scheint, so legt die Masse der Dorfbewohner den {1!role:bold} {0:@} um.",
+        "Unter viel Lärm lynchen die Pechgabel-lagerischen Dorfbewohner {0:@}, was sich als... {1!role:article} {1!role:bold} entpuppte.",
```

Comment:
not sure there's an idiomatic way to put this, might remove it?

Reply:
i has a another solution see common upload but its not ready yet